src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int countOfBalls = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[countOfBalls];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Lottery().getRandomBall();
+            System.out.println("№" + (i + 1) + "-" + balls[i]);
+        }
```

Comment:
1) Please, pay attention to the task description. You don't need to store balls anywhere
2) Do you really need to create a new lottery each time you want to create a new ball?
3) Minor: I suggest just passing a newly generated ball to the println() method and letting to overridden toString() of the Ball class do all the work))